<?xml version="1.0" encoding="UTF-8"?>
<testrun name="ex.practice.fizzbuzz.FizzBuzzTest.getFizzBuzzRandomNumberメソッドでランダム数のジェネレータを指定した場合.FizzBuzzの対応文字列を出力している の再実行" project="JUnitPractice" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testcase name="FizzBuzzの対応文字列を出力している" classname="ex.practice.fizzbuzz.FizzBuzzTest.getFizzBuzzRandomNumberメソッドでランダム数のジェネレータを指定した場合" time="0.0">
    <error>org.junit.experimental.theories.internal.ParameterizedAssertionError: FizzBuzzの対応文字列を出力している(Fizzの組み合わせが正しいこと)
	at org.junit.experimental.theories.Theories$TheoryAnchor.reportParameterizedError(Theories.java:183)
	at org.junit.experimental.theories.Theories$TheoryAnchor$1$1.evaluate(Theories.java:138)
	at org.junit.experimental.theories.Theories$TheoryAnchor.runWithCompleteAssignment(Theories.java:119)
	at org.junit.experimental.theories.Theories$TheoryAnchor.runWithAssignment(Theories.java:103)
	at org.junit.experimental.theories.Theories$TheoryAnchor.runWithIncompleteAssignment(Theories.java:112)
	at org.junit.experimental.theories.Theories$TheoryAnchor.runWithAssignment(Theories.java:101)
	at org.junit.experimental.theories.Theories$TheoryAnchor.evaluate(Theories.java:89)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
Caused by: java.lang.AssertionError: 
Expected: is &lt;ex.practice.fizzbuzz.FizzBuzz$RtnData@44a613f8&gt;
     got: &lt;ex.practice.fizzbuzz.FizzBuzz$RtnData@736921fd&gt;

	at org.junit.Assert.assertThat(Assert.java:780)
	at org.junit.Assert.assertThat(Assert.java:738)
	at ex.practice.fizzbuzz.FizzBuzzTest$getFizzBuzzRandomNumberメソッドでランダム数のジェネレータを指定した場合.FizzBuzzの対応文字列を出力している(FizzBuzzTest.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.experimental.theories.Theories$TheoryAnchor$2.evaluate(Theories.java:167)
	at org.junit.experimental.theories.Theories$TheoryAnchor$1$1.evaluate(Theories.java:133)
	... 20 more

</error>
  </testcase>
</testrun>
